circle.yml

```diff
@@ -22,6 +23,11 @@ test:
   # Grep error on the documentation
   override:
     - cat ~/log.txt && if grep -q "Traceback (most recent call last):" ~/log.txt; then false; else true; fi
+deployment:
+  push:
+    branch: master
+    commands:
+      - bash continuous_integration/push_doc.sh
```

Comment:
Yes it means that it won't deploy on any other branch than master. I tried it on my own repository. I can add a safety but I don't think it's required.